### ISSUE TITLE
make chrome-build failed

### ISSUE THREAD

**Issue by WellingtonSI** [lwthiker/curl-impersonate#216]
I follow all the native build steps, but when I get to the moment to run the "make chrome-build" command I get this error "make: *** No rules for creating the 'chrome-build' target. Stop." . I run this command inside the build folder, following the command order and I understood that it would be in that location.

**Comment by bashonly** [comment]
did you run `../configure` first?

**Comment by WellingtonSI** [comment]
yes, I ran the ../config command first and the only error I got was "configure: error: failed to find zlib"

**Comment by yifeikong** [comment]
Then you need to make sure that `configure` runs successfully. The `configure` script generates the `Makefile`.

For `zlib`, consider using brew or apt to install libz on your platform.

**Comment by WellingtonSI** [comment]
If the `Makefile` script is generated inside the Build folder where the `configure` command was made, then it was unsuccessful. What could cause the failure and would there be a message indicating?

**Comment by lwthiker** [comment]
@WellingtonSI Can you provide more details on the errors you face, including the commands that you ran and their output? Right now it is difficult to understand exactly the problem you are facing.

**Comment by phapntm** [comment]
i have the same issue then i realize that i need install firefox first although i only need chrome.

**Comment by waschinski** [comment]
`apt install zlib1g-dev` did work for me on Debian.
